packages/common/src/ee/embed/index.ts

```diff
@@ -101,7 +102,7 @@ type CommonEmbedJwtContent = {
     isPreview?: boolean;
     dashboardFiltersInteractivity?: {
         enabled: FilterInteractivityValues | boolean;
-        allowedFilters?: string[];
+        allowedFilters?: string[] | null;
```

Comment:
so now we accept undefined | null | [] , right ?

Reply:
That's right. The problem was that python serializes `None` to `null` rather than `undefined`. In our docs, we tell Python users for set `allowedFilters: None`. So Python users have been always failing validation, but we haven't been enforcing the schema until I "fixed" it 😅